.github/python-app.yml

```diff
@@ -0,0 +1,57 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
```

Comment:
the file should be in `.github/workflows/python-app.yml`, else you can see it does not actually run.